OpenMEEG/Wrapping/src/test_python.py

```diff
@@ -159,7 +159,7 @@
 # For a Vector
 v=hm(1,10,1,1).getcol(0)
```

Comment:
no, no wait ;-/ can we say i did not touch this file ? ;-)
(do you want a very big PR with everything ??, I was more about to split by subjects as much as I can)